src/features/components/atoms/card/card-primary/index.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import Typography from '../../typography'
+import Flex from '../../flex-box'
+import Link from '../../link'
+import Image from '../../image'
+import { Localize } from 'components/localization'
+import { TString } from 'types/generics'
+import dclsx from 'features/utils/dclsx'
+import useBreakpoints from 'components/hooks/use-breakpoints'
+
+type CardItemsProps = {
+    header: TString
+    description: TString
+    icon?: string
+    link?: `/${string}`
+}
+
+const CardPrimary = ({ header, description, icon, link }: CardItemsProps) => {
+    const { is_mobile_or_tablet } = useBreakpoints()
+
+    return (
+        <Flex.Box
+            className={dclsx('item_container')}
+            direction="col"
+            gap="4x"
+            radius="8x"
+            lg={{ padding: '12x' }}
+            padding="8x"
+        >
+            {icon && <Image src={icon} width={48} height={48} pb="4x" lg={{ pb: '8x' }} />}
```

Comment:
Hi @meenakshi-deriv I saw another component of icon. Can we use that component here. 
Thanks @meenakshi-deriv 🙏

Reply:
Hi @fasihali-deriv , which is that component?